### PR TITLE
[#2130] fix(server): Potential huge partition limit invalid when flushing is slow

### PR DESCRIPTION
**Pull request by zuston**
### What changes were proposed in this pull request?

fix potential huge partition limit invalid when flushing is slow

### Why are the changes needed?

Fix: #2130

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Unit tests

### CODE REVIEW

**Comment by zuston** [comment]
PTAL @maobaolong

**Review by maobaolong — COMMENTED** [review]
@zuston Thanks to improve this, LGTM for this PR, left a comment in line.

**Review by maobaolong — APPROVED** [review]

File: server/src/main/java/org/apache/uniffle/server/HugePartitionUtils.java [hunk 1]
```diff
@@ -130,11 +131,9 @@ public static boolean limitHugePartition(
       int partitionId,
```
[the remark this answers is not in the document]
Comment (zuston) [reply server/src/main/java/org/apache/uniffle/server/HugePartitionUtils.java:131]: Yes, this will reject the all partitionIds for one spark task.

**Comment by zuston** [comment]
cc @jerqi PTAL

**Review by jerqi — APPROVED** [review]

**Pull request merged**